offerImport/importHandler/offerModel.go

```diff
@@ -0,0 +1,52 @@
+package importHandler
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+)
+
+type Offer struct {
+	ID        string
+	Name      string
+	Receiver  string
+	ValidFrom time.Time
+	ExpiresAt time.Time
+	Countries []string
+}
+
+func newOffer(item map[string]interface{}) *Offer {
+	offer := Offer{
+		ID:       fmt.Sprintf("%v", item["offerKey"]),
+		Name:     fmt.Sprintf("%v", item["name"]),
+		Receiver: fmt.Sprintf("%v", item["buyerId"]),
+	}
+
+	if item["startDate"] != nil {
+		offer.ValidFrom = convertMillisecondsToTime(item["startDate"].(float64))
```

Comment:
Why float64 is used here? Unix time is long.

Reply:
it's interface and it should be converted to string or float, otherwise:
interface conversion: interface {} is float64, not int64

moved type convertation to one point